dev/io.openliberty.webcontainer.servlet.6.0.internal_fat/publish/servers/servlet60_xPoweredByHeaderTest/server.xml

```diff
@@ -8,12 +8,15 @@
     Contributors:
         IBM Corporation - initial API and implementation
  -->
-<server description="Server for testing Webcontainer">
+<server description="Server for test disableXPoweredBy configuration.r">
 
     <include location="../fatTestPorts.xml"/>
 
     <featureManager>
         <feature>servlet-6.0</feature>
     </featureManager>
+    
+     <webContainer disableXPoweredBy="false"/>
```

Comment:
Should this be here? Or are you making sure that servlet 6.0 ignores this setting? Can you add a comment?

Reply:
I've added a comment in the code review commit that I just pushed to this PR. This server.xml is used in the `test_XPoweredByCanNotBeEnabled` test.